data-service-client/src/main/java/com/amazonaws/blox/dataservice/client/v1/model/InstanceGroup.java

```diff
@@ -0,0 +1,17 @@
+package com.amazonaws.blox.dataservice.client.v1.model;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+import java.util.List;
+
+@Value
+@Builder
+public class InstanceGroup {
+
+    @NonNull
+    private String clusterArn;
+
+    private List<Attribute> attributes;
```

Comment:
Is attribute ordering or allowing duplicates important? Otherwise a Map or Set might be a better fit here.